applications/next-visit-fan-out/values.yaml

```diff
@@ -2,6 +2,7 @@ knative:
   hscUrl: 'http://prompt-proto-service.prompt-proto-service-hsc/next-visit'
   latissUrl: http://prompt-proto-service.prompt-proto-service-latiss/next-visit
   lsstcomcamUrl: http://prompt-proto-service.prompt-proto-service-lsstcomcam/next-visit
+  lsstcomcamsimUrl: http://prompt-proto-service.prompt-proto-service-lsstcomcamsim/next-visit
   lsstcamUrl: http://prompt-proto-service.prompt-proto-service-lsstcam/next-visit
```

Comment:
I don't think I understand these URLs. Are they really _exactly the same_ for `usdfdev` and `usdfprod`?

If they're supposed to be the same, then I suggest removing those blocks from the per-environment files.

Reply:
You are right. They are exactly the same in `usdfdev` and `usdfprod`. I'm removing this from the per-environment files.